pkg/app/pipectl/cmd/application/delete.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2020 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package application
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/pipe-cd/pipecd/pkg/app/server/service/apiservice"
+	"github.com/pipe-cd/pipecd/pkg/cli"
+)
+
+type delete struct {
+	root *command
+
+	appID string
+}
+
+func newDeleteCommand(root *command) *cobra.Command {
+	c := &delete{
+		root: root,
+	}
+	cmd := &cobra.Command{
+		Use:   "delete",
+		Short: "Delete application.",
+		RunE:  cli.WithContext(c.run),
+	}
+
+	cmd.Flags().StringVar(&c.appID, "app-id", c.appID, "The application ID.")
```

Comment:
nits, the flag name can be `id`, but it's up to you 👍

Reply:
Thanks for the code review!
I think ```app-id``` flag is better than ```id``` flag, because we can see at a glance what the id is and the flag of ```pipectl application get``` is also ```app-id```, so it's consistent.

Reply:
Nice catch, then lets keep it as is 👍